project-chip/connectedhomeip
# Fix Telink Docker image instability

#### Problem
* Fixes https://github.com/project-chip/connectedhomeip/issues/8640

#### Change overview
Instability occurs due to Zephyr repo was not versioned. Now Zephyr version is specified as v2.5.0

#### Testing
Tested manually. Steps:
1. build new docker image
2. run **$ ./scripts/examples/telink_example.sh lighting-app tlsr9518adk80d** in the docker container
3. check that build was successful
